dfkernel/ipkernel.py

```diff
@@ -50,6 +59,10 @@ def __init__(self, **kwargs):
         self.shell.display_pub.get_execution_count = lambda: int(
             self.execution_count, 16
         )
+
+        get_ipython().kernel.comm_manager.register_target('dfpackages', self.dfpackages_comm)
+        get_ipython().kernel.comm_manager.register_target('dfconvert', self.dfconvert_comm)
```

Comment:
Please just register this as a single comm channel, I don't see any reason why this should be two comm channels, it also helps explain what this comm channel is being used for (dfconvert)